<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatientDialog</class>
 <widget class="QWidget" name="PatientDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>  <!-- Augmenter la largeur si nécessaire -->
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Patients</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBoxPatientForm">
     <property name="title">
      <string>Formulaire Patient</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelNom">
        <property name="text">
         <string>Nom :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditNom"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelPrenom">
        <property name="text">
         <string>Prénom :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditPrenom"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelDateNaissance">
        <property name="text">
         <string>Date de Naissance :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDateEdit" name="dateEditNaissance"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelTelephone">
        <property name="text">
         <string>Téléphone :</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEditTelephone"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelAdresse">
        <property name="text">
         <string>Adresse :</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QTextEdit" name="textEditAdresse"/>
      </item>
      <item row="5" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btnAjouter">
          <property name="text">
           <string>Ajouter</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnModifier">
          <property name="text">
           <string>Modifier</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSupprimer">
          <property name="text">
           <string>Supprimer</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidgetPatients">
     <property name="columnCount">
      <number>6</number>  <!-- 6 colonnes, y compris Adresse -->
     </property>
     <property name="horizontalHeaderLabels">
      <stringlist>
       <string>ID</string>
       <string>Nom</string>
       <string>Prénom</string>
       <string>Date de Naissance</string>
       <string>Téléphone</string>
       <string>Adresse</string>  <!-- Nouvelle colonne -->
      </stringlist>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
